ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -24,16 +24,17 @@ def get_queue_manager_connection(config, logger):
         # By testing with a normal connection first, we avoid making unnecessary SSL connections.
         # This does not fix the memory leak but mitigate its likelihood.
         # Details: https://github.com/dsuch/pymqi/issues/208
-        try:
-            get_normal_connection(config, logger)
-        except pymqi.MQMIError as e:
-            logger.debug(
-                "Tried normal connection before SSL connection to ensure channel exists."
-                "This is expected to fail with SSL or host unavailable errors. It failed with: %s",
-                e,
-            )
-            if e.reason == pymqi.CMQC.MQRC_UNKNOWN_CHANNEL_NAME:
-                raise
+        if config.try_normal_connection:
+            try:
+                get_normal_connection(config, logger)
+            except pymqi.MQMIError as e:
+                logger.debug(
+                    "Tried normal connection before SSL connection to ensure channel exists."
```

Comment:
If you decide to change the option name, this log should also be updated to use the new phrasing.